build/files.js

```diff
@@ -280,24 +276,15 @@ module.exports['_stream_writable.js'] = [
 
 module.exports['internal/streams/buffer_list.js'] = [
     [
-      /(?:var|const) (?:{ )Buffer(?: }) = require\('buffer'\)(?:\.Buffer)?;/,
+      /const \{ inspect \} = require\('util'\);/,
       `
-const Buffer = require('buffer').Buffer
-const util = require('util')
+const { inspect } = require('util')
+const custom = inspect && inspect.custom || 'inspect'
```

Comment:
Won't this browserify all of `util`? I wrote https://github.com/mafintosh/inspect-custom-symbol to get around that (util is pretty big!)

Reply:
@mafintosh that module uses the symbol instead of `'inspect'` as a key. See https://github.com/mafintosh/inspect-custom-symbol/blob/master/index.js#L4. @BridgeAR should we use the symbol or the property here?

Reply:
Also see #356 to remove the use of util.

Reply:
It should be `'inspect'`. Otherwise old Node.js versions won't work and a symbol has no other effect on browserify in this case than the string.

I don't use browserify often enough to know exactly what it would pull in but it should only be the parts that are used and in this case it is only the inspection property. So that should be fine.

Reply:
I think we should land this as is and deal with removing util in #356. What do you think? May I get a couple of LGTM?

Reply:
@BridgeAR it pulls in the entire `util` source. that's why people use the inherits module instead of util.inherits on npm also :)

@mcollina Ya, solving it elsewhere is fine by me